mithril-common/src/crypto_helper/cardano/key_certification.rs

```diff
@@ -252,7 +252,7 @@ impl KeyRegWrapper {
             if cfg!(not(feature = "allow_skip_signer_certification")) {
                 Err(ProtocolRegistrationErrorWrapper::OpCertMissing)?
             }
-            println!("WARNING: Uncertified signer regsitration by providing a Pool Id is deprecated and will be removed soon! (Pool Id: {:?})", party_id);
+            println!("WARNING: Uncertified signer regsitration by providing a Pool Id is decommissionned and must be used for tests only! (Pool Id: {:?})", party_id);
```

Comment:
Should we put this warning directly when the signers are initialised? So, rather than warn users only when they've tried to register, if they already run the node with the "allow_skip_signer_certification", then this warning should appear, saying they won't be able to register. WDYT?

Reply:
It's difficult to know that at signer initialization because the feature flag `allow_skip_signer_certification` is applied to `mithril-common`. But I agree with you that this could be a good option :thinking:

Reply:
ah, good point. We could put the warning with `StmInitializerWrapper::setup()` then. WDYT?

Reply:
Yes, I moved the warning in `StmInitializerWrapper::setup()` in https://github.com/input-output-hk/mithril/pull/653/commits/b83300e0a7cc177dda7f7f3a2462c758aad4e482